src/stories/Library/nav-grid/init-nav-grid.js

```diff
@@ -13,6 +13,15 @@ window.addEventListener("load", () => {
 
     button.addEventListener("click", () => {
       grid.classList.remove("nav-grid--folded");
+
+      // The 7th item is the first one that is initially hidden.
+      // For more details, refer to the ".nav-grid--folded" class in design-system src/stories/Library/nav-grid/nav-grid.scss
+      const seventhNavGridItem = grid.querySelector(
+        ".nav-grid__item:nth-child(7)"
```

Comment:
This is waaay too magical.
Instead of looking for the 7th, you should instead try to find the first one that is hidden or something :)

Reply:
How can I do it when the display none logic is not set on the items but on the container?

```
.nav-grid--folded {
  .nav-grid__item:nth-child(n + 7) {
    display: none;
  }

Reply:
You could either check the CSS:

https://stackoverflow.com/questions/4866229/check-element-css-display-with-javascript

Or, check if it is visible:

https://stackoverflow.com/a/21696585

Reply:
Thanks 🙏